src/section-solving-system-equations-by-graphing.mbx

```diff
@@ -317,76 +317,115 @@
+
+            <figure>
+                <caption>Graphs of <m>y=2x-4</m> and <m>6x-3y=12</m></caption>
+                <image width="50%">
+                    <description></description>
+                    <latex-image-code>
+                    <![CDATA[\begin{tikzpicture}
+                        \begin{axis}[
+                                    legend cell align=left, legend pos=north west,
+                                    ]
+                            \addplot[primarycurve,domain = -1.5:5.5]{2*x-4.1};
+                            \addplot[secondarycurve,domain = -1.5:5.5]{2*x-4};
+                            \addplot[soliddot,color=firstcolor] coordinates {(0,-4.1) (1,-2.1) (2,-0.1)};
+                            \addplot[soliddot,color=secondcolor] coordinates {(0,-4) (1,-2) (2,0)};
+                            \legend{$y=2x-4$,$6x-3y=12$}
+                        \end{axis}
+                    \end{tikzpicture}]]>
+                    </latex-image-code>
+                </image>
+            </figure>
+
+            As we can see these were actually the <em>same</em> line. To determine the solution to this system, we'll note that they overlap everywhere on each line. This means that we have an infinite number of solutions: all points that fall on each line. To write this solution set, we need to write a set representing all points on the line. We can do so like this: <m>\{(x,y)\mid y=2x-4\}</m>. 
+            </p>
+        </example>
+
+        <remark><p>
+            What would have happened if we had instead decided to solve <m>6x-3y=12</m> for <m>y</m> in 
+            <xref ref="example-solving-systems-by-grahping-equivalent-equations">Example</xref>? Doing so, we'd get this:
+                <md>
+                    <mrow>6x-3y\amp=12</mrow>
+                    <mrow>6x-3y\subtractright{6x}\amp=12\subtractright{6x}</mrow>
+                    <mrow>-3y\amp=-6x+12</mrow>
+                    <mrow>\multiplyleft{-\frac{1}{3}}(-3y)\amp=\multiplyleft{-\frac{1}{3}}(-6x+12)</mrow>
+                    <mrow>y\amp=2x-4</mrow>
+                </md>
+            As we can see, this is the exact same equation as the other equation in our system. This shows that these two equations are equivalent equations. Any time we try to solve a system of equations where the equations are equivalent, we'll have an infinite set of solutions.
+        </p></remark>
+
+
+        <p>Let's summarize the possible solutions we can have to a system of linear equations:<ul>
```

Comment:
Would this `ul` be better as a `dl`? Things like "Parallel lines:" always raise this question in my mind.